packages/ceramic-cli/src/ceramic-daemon.ts

```diff
@@ -16,53 +19,73 @@ class CeramicDaemon {
       next()
     })
     this.app.post(toApiPath('/create'), this.createDoc.bind(this))
-    this.app.get(toApiPath('/show/ceramic/:doctype/:cid'), this.show.bind(this))
-    this.app.get(toApiPath('/state/ceramic/:doctype/:cid'), this.state.bind(this))
-    this.app.post(toApiPath('/change/ceramic/:doctype/:cid'), this.change.bind(this))
+    this.app.get(toApiPath('/show/ceramic/:cid'), this.show.bind(this))
+    this.app.get(toApiPath('/state/ceramic/:cid'), this.state.bind(this))
+    this.app.post(toApiPath('/change/ceramic/:cid'), this.change.bind(this))
     const server = this.app.listen(7007, () => {
```

Comment:
We can put a daemon `port` in the configuration file

Reply:
Yep, that should be configurable once we have a config file!